lesson_01/index.pt-br.md

```diff
@@ -0,0 +1,218 @@
+**Lição Um de Linguagem Assembly do FFmpeg**
+
+**Introdução**
+
+Bem-vindo à Escola de Linguagem Assembly do FFmpeg. Você deu o primeiro passo na jornada mais interessante, desafiadora e gratificante da programação. Estas lições fornecerão os fundamentos sobre como assembly é usado no FFmpeg e abrirão seus olhos para o que realmente acontece dentro do seu computador...
+
+**Conhecimentos Necessários**
+
+* Conhecimento em C, especialmente ponteiros. Se você não está familiarizado com C, você encontrará todos os fundamentos no livro [The C Programming Language](https://en.wikipedia.org/wiki/The_C_Programming_Language).
+* Matemática do ensino médio (escalar vs vetor, adição, multiplicação, etc...)
+
+**O que é Assembly?**
+
+Assembly é uma linguagem de programação onde o código que você escreve corresponde diretamente a instruções compreensíveis por uma CPU. O assembly legível por humanos é, como o nome indica, *montado* em dados binários, conhecido como *linguagem de máquina*, que a CPU pode entender. O código assembly é frequentemente chamado de "assembly" ou "asm" abreviadamente.
```

Comment:
Não valeria a pena adicionar uma nota da tradução que "to assemble" em inglês significa "montar"? Caso contrário, o trecho que diz "como o nome indica" não faz sentido.

Wouldn't it be worth it to add a translation note mentioning that "to assemble" in english is "montar" in portuguese? Otherwise, the part saying "como o nome indica" (as its name hints) makes no sense.